src/components/ui/Toast/stories/Toast.stories.tsx

```
@@ -0,0 +1,279 @@
+// Preloaded — 3 toasts visible on load
+// ─────────────────────────────────────────────────────────────────────────────
+
+function PreloadedDemo() {
+    useCleanup();
+    const manager = Toast.useToastManager();
+
+    useEffect(() => {
+        setTimeout(() => manager.add({ title: 'Third — oldest', variant: 'default' }), 0);
+        setTimeout(() => manager.add({ title: 'Second notification', variant: 'info' }), 80);
+        setTimeout(() => manager.add({ title: 'First — newest', variant: 'success' }), 160);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)